tests/bluetooth/tester/src/bttester.h

```diff
@@ -594,6 +594,7 @@ struct gatt_read_long_rp {
 } __packed;
 
 #define GATT_READ_MULTIPLE		0x14
+#define GATT_READ_MULTIPLE_VAR		0x20
```

Comment:
I kind of get why this is placed here, but I feel it would still be better to place it in an ordered manner.

Reply:
Moved to the bottom.